webpack/scenes/Subscriptions/Manifest/SimpleContentAccess.js

```diff
@@ -21,6 +22,14 @@ const SimpleContentAccess = (props) => {
     }
   };
 
+  const simpleContentAccessText = () => {
+    if (!simpleContentAccessEligible && !isSimpleContentAccessEnabled) {
+      return __('Simple Content Access has been disabled by the upstream organization administrator.');
```

Comment:
While the API request is in flight, this message is shown.  I think it would make more sense to default to the other message instead.  (I know it's only half a second but I feel like details like this make for a better user experience.)

This will require uncoupling the enablement of the switch and the message that's shown.